### DISCUSSION TITLE
Local file read and write for conversion

### DISCUSSION THREAD

**Discussion by MugdhaHardikar-GSLab** [gotenberg/gotenberg#772] [Q&A] [answered]
I want to convert lots of (100s of files) into pdf using gotenberg. I am deploying the docker container locally on my system. As my files are big sending them over a REST API is taking too much of a time. 
Is there any API exposed by gotenberg where I can pass the source and target path in API and then gotenberg will read the file from there and then write the converted file to the specified destination. 
I have mounted the volumn on the container so I know the paths.

**Comment by gulien** [comment, marked as answer]
Hello @MugdhaHardikar-GSLab,

Gotenberg only handles files over HTTP. 

Depending on your requirements, you may use LibreOffice (via https://github.com/unoconv/unoserver for instance) or write your own script using https://pptr.dev/.

**Comment by MugdhaHardikar-GSLab** [comment]
Thanks a lot for the quick reply and a suggestion to move forward.
